client/app/telecom/overTheBox/remote/overTheBox-remote.html

```diff
@@ -17,92 +17,102 @@ <h2 class="widget-presentation-title" data-translate="overTheBox_remote_configur
                         </tr>
                     </thead>
                     <tbody data-ng-if="OverTheBoxRemote.remotes.length === 0">
-                        <tr data-ng-show="OverTheBoxRemote.loading">
+                        <tr data-ng-if="OverTheBoxRemote.loading">
                             <td colspan="5" class="text-center">
                                 <spinner></spinner>
                             </td>
                         </tr>
-                        <tr data-ng-hide="OverTheBoxRemote.loading">
+                        <tr data-ng-if="!OverTheBoxRemote.loading">
                             <td colspan="5" data-translate="overTheBox_remote_table_empty" class="text-center"></td>
                         </tr>
                     </tbody>
                     <tbody data-ng-repeat="remote in OverTheBoxRemote.paginatedRemotes track by remote.remoteAccessId">
                         <tr>
-                            <td>
-                                <button type="button" class="btn btn-link" data-ng-click="remote.show = !remote.show">
-                                    <i class="glyphicon" data-ng-class="{'glyphicon-chevron-down':remote.show, 'glyphicon-chevron-right': !remote.show}"></i>
-                                    <span data-ng-bind="remote.remoteAccessId"></span>
+                            <th scope="row">
+                                <button type="button"
+                                        class="btn btn-link"
+                                        data-ng-click="remote.show = !remote.show">
+                                        <span class="ovh-font fs12"
+                                              data-ng-class="{'ovh-font-disclosure-down':remote.show, 'ovh-font-disclosure-right': !remote.show}"
+                                              aria-hidden="true">
+                                        </span>
+                                        <span data-ng-bind="remote.remoteAccessId"></span>
                                 </button>
-                            </td>
+                            </th>
                             <td>
                                 <span>{{ ('overTheBox_status_' + remote.status) | translate}}</span>
-                                <spinner type="volume" class="left-space-m4" data-ng-show="remote.status !== 'active'"></spinner>
+                                <spinner type="volume" class="left-space-m4" data-ng-if="remote.status !== 'active'"></spinner>
```

Comment:
Are you sure of using `type="volume"` attribute? Not yours but I guess it could be removed.